client/python/gradio_client/client.py

```diff
@@ -772,6 +786,12 @@ def status(self) -> StatusUpdate:
                 )
             else:
                 with self.communicator.lock:
+                    eta = self.communicator.job.latest_status.eta
+                    if self.verbose and self.space_id and eta and eta > 0:
```

Comment:
I think `eta > 0` is too loose of a condition, no?

Reply:
Oh sorry that was supposed to >30 lol

Reply:
Changed it during test, will revert